readme.md

```diff
@@ -16,24 +16,40 @@ https://github.com/cosmos/relayer.git
 cd relayer
 make install
 
-# Bootstrap two local chains
-make start-dev
-
-# Use relayer to link the chains
-make start-rly
+# Bootstrap two local chains & start the relayer with development data
+make init
 ```
 
-### Demo
+### Send Asset Demo
 
 ```bash
+# These address are defined in init.sh for development purposes
+VAL_1=cosmos1mjk79fjjgpplak5wq838w0yd982gzkyfrk07am
+VAL_2=cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs
+
 # Register an IBC Account on chain test-2 
-icad tx intertx register ibcaccount channel-0 --from val1 --chain-id test-1 --gas 90000 --home ./data/test-1 --node tcp://localhost:16657
+icad tx intertx register --from val1 --source-port ibcaccount --source-channel channel-0 --chain-id test-1 --gas 90000 --home ./data/test-1 --node tcp://localhost:16657
+
+# Get the address of interchain account
+icad query intertx ibcaccount $VAL_1 ibcaccount channel-0 --node tcp://localhost:16657
```

Comment:
is chain-id not needed here?

Reply:
guess not for queries

Reply:
We are targetting the node directly which is chain `test-1` in this case. `--node tcp://localhost:16657 `

It works for me. Is there an added benefit with `chain-id`? I guess this is just for local testing/development.